### ISSUE TITLE
add fixed footer with github icon, project version and link to the project

### ISSUE THREAD

**Issue by dawidmalina** [erwindon/SaltGUI#149]
Will you be interested in fixed footer which will contains (content same as on the login page):
- github icon
- saltgui version
- link to the project

It will be look like this:
![zrzut ekranu 2019-01-26 o 21 06 05](https://user-images.githubusercontent.com/5910851/51792185-38a6ea80-21ae-11e9-86d0-0039407022d5.png)

**Comment by erwindon** [comment]
No. That is only helpful (and then only a little bit) for SaltGUI developers, not for users.
